packages/components/src/tabs/styles.ts

```diff
@@ -13,12 +13,31 @@ import { space } from '../utils/space';
 import { reduceMotion } from '../utils/reduce-motion';
 
 export const TabListWrapper = styled.div`
+	position: relative;
 	display: flex;
 	align-items: stretch;
 	flex-direction: row;
 	&[aria-orientation='vertical'] {
 		flex-direction: column;
 	}
+	&::after {
+		content: '';
+		position: absolute;
+		left: var( --indicator-left );
+		bottom: 0;
+		width: var( --indicator-width );
+		height: 0;
+		border-bottom: var( --wp-admin-border-width-focus ) solid
+			${ COLORS.theme.accent };
+		pointer-events: none;
+		@media not ( prefers-reduced-motion: reduce ) {
+			transition: left 0.2s ease-out;
```

Comment:
It think we can make the focus-visible ring transition a bit more congruent with the new transitions. It feels too abrupt.

https://github.com/WordPress/gutenberg/blob/ecf24f9b8adc4147ef0b50e7bcfec3f4187f7351/packages/components/src/tabs/styles.ts#L92-L103

☝️ Switching this to a transition between opacity 0 and 1 feels better to me, rather than the current `transition: all` that transitions the entire box-shadow in a clunky way. What do you think?

Reply:
Done. Feels much better indeed 👍